features/device-config/rest/pom.xml

```diff
@@ -86,16 +80,26 @@
             <groupId>org.opennms.core.test-api</groupId>
             <artifactId>org.opennms.core.test-api.services</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest</artifactId>
+            <version>2.2</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.opennms.features.device-config</groupId>
+            <artifactId>org.opennms.features.device-config.api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
         <dependency>
```

Comment:
test dependencies should be grouped together below main dependencies

Reply:
Fixed